docs/rfcs/0005-suggested-excluded-apps-tech-spec.md

```diff
@@ -0,0 +1,154 @@
+
+Other options considered
+------------------------
+
+Design and research went through several iterations, so the [designs](https://www.figma.com/file/UZYzma7hlcfE5ke3z8jGbN/App-exclusions-suggestions?type=design&node-id=196-6366&mode=design&t=RL1hdfBQLMS1rKVa-0) were set before this tech spec was written. (See [completed tickets in the epic](https://mozilla-hub.atlassian.net/browse/VPN-4412) for more background.)
+
+Other options considered included dynamically serving the suggested lists from addons or our repo or Guardian, each of which added complexity for hosting and validating the format. With the rarity of the list being updated (and lack of time-sensitivity when it does), this draft bundles the lists with the app. Hosting the list in Guardian was difficult (which is a private repo, so the list would need to be hosted elsewhere to allow public suggestions).
+
+### Other options considered for suggested app files
+
+The goals for the file format:
+-   Easy to read by humans
+-   Easy for humans to add to, no matter their level of technical proficiency
+-   Easy for us to parse in the app
+-   Allowing comments in the file.
+
+Initially, there was one additional goal (no longer consideration):
+-   To have one file for all platforms, so that we'd remember to add a specific app to all platforms at once. With a conversation around potentially wanting different suggested app categories on mobile, this seems unimportant. (However, if one file per platform seems more streamlined, it's still acceptable.)
+
+Other options considered included CSV, YAML, and plain text files. While YAML could be useful (as it's easy for less-technical people to read), we don't have a good YAML parser in the app yet. We could add a library, of course. However, we'd need to write our own document format test, because the YAML file (or CSV) would need to be structured in a specific way. 
+
+Ultimately, JSON version seemed simpler (and thus easier to build) - there aren't the same parsing concerns and it's easy for contributors.
+
+Metrics
+-------
+
+We will create several metrics as part of this work; they are described in the appropriate spots in the "Implementation" section.
+
+We'd expect the `apps_excluded` metric to increase after this is implemented on platforms which have app exclusions. Both the number of people with more than 0 apps excluded and the average count of apps excluded should increase.
+
+If Nimbus is available by the time this feature is complete, we will use it to roll out the test.
```

Comment:
Given the unknowns about the long-term shape of some aspects of this feature I think we should stipulate that it be rolled out as a Nimbus experiment. Nimbus availability also means we'd have the ability to disable it without a client update.

Reply:
Good call, updated.

(We'll also be able to disable by turning off the feature.)